src/clr-angular/utils/i18n/common-strings.interface.ts

```diff
@@ -85,4 +85,12 @@ export abstract class ClrCommonStrings {
    * Datagrid: pick columns
    */
   pickColumns?: string;
+  /**
+   * Datagrid numeric filter: min
+   */
+  min?: string;
```

Comment:
When you rebase onto `master`, we'll need to add these into the internationalization documentation as well.

Reply:
Keeping open for the internationalization todo.